### PR TITLE
VOLDEV-108: Make RecentChanges js object public

### PR DESCRIPTION
**Pull request by Kangaroopower**
@Grunny @timmyquivy 

This is a voldev ticket but I'm not actually voldev so apologies if this is not the way to do a PR. I made the RecentChanges class public because for the past year or so AjaxRC has been using a duplicate version of this class that's in the script itself to make sure that the Dropdown is reinitialized every time that the refresh is run. This means though that the code on the script could go out of date at any time and that it's running an exact duplicate of the code that Wikia made, which is never really efficient or a good thing.

I added RecentChanges to the Wikia global in this patch- it can be found under Wikia.RecentChanges. It's not camel case because the original private object wasn't and anyways theres a precedent of noncamel case stuff under the Wikia global (LazyQueue).

Sorry for the huge explanation for a two line change.

### CODE REVIEW

**Comment by wladekb** [comment]
@Grunny ping

**Comment by onei** [comment]
@Kangaroopower LGTM, but perhaps it could have a note about it being used by AjaxRC? If someone changes the name of it (or undoes this unintentionally) it could cause errors on hundreds of wikis and would likely not be picked up during testing.

**Comment by Kangaroopower** [comment]
Ok I can do that

File: extensions/wikia/RecentChanges/js/RecentChanges.js [hunk 1]
```diff
@@ -41,4 +44,5 @@ jQuery(function($) {
 	};
 
 	RecentChanges.init();
+	Wikia.RecentChanges = RecentChanges; //needed for AjaxRC
```
Comment (Grunny) [inline extensions/wikia/RecentChanges/js/RecentChanges.js]: I'd use `window.Wikia.RecentChanges` here too.

**Comment by Kangaroopower** [comment]
@Grunny done

**Comment by Kangaroopower** [comment]
So is this going in with tomorrow's system update?

**Pull request merged**